src/detection/localip/localip_linux.c

```diff
@@ -128,8 +132,12 @@ const char* ffDetectLocalIps(const FFLocalIpOptions* options, FFlist* results)
 
     for (struct ifaddrs* ifa = ifAddrStruct; ifa; ifa = ifa->ifa_next)
     {
-        if (!ifa->ifa_addr || !(ifa->ifa_flags & IFF_RUNNING))
+        if (!ifa->ifa_addr)
             continue;
+#ifdef IFF_RUNNING
+        if (!(ifa->ifa_flags & IFF_RUNNING))
```

Comment:
How can we know if it's running then?

Reply:
AFAIK this is not really used in most drivers on Linux anyway, it's here for BSD compatibility.

I'll add our own extra flags that I missed (IFF_LINK,…CONFIGURING…).